.github/workflows/support-packages.yml

```diff
@@ -68,6 +68,25 @@ jobs:
             /tmp/support-packages/output \
             gs://${{ inputs.downstream-drivers-bucket }}
 
+      - name: Check empty support packages
+        id: empty-check
+        run: |
+          empty=()
+          for mod_ver_dir in /tmp/support-packages/output/*; do
+            mod_ver="$(basename "$mod_ver_dir")"
+            if [[ -e "$mod_ver_dir/empty" ]]; then
+              empty+=("$mod_ver")
+              rm -rf "$mod_ver_dir"
+            fi
+          done
+
+          if ((${#empty[@]})); then
+            echo 2>&1 "Found empty support packages for module versions: ${empty[*]}"
+            echo "has-empty-packages=true" >> "$GITHUB_OUTPUT"
```

Comment:
Instead of using an output var, we can probably do this check in the step that starts in line 146 and directly exit with an error. This only works if you apply my other comment on using a single `empty` file, rather than one per version.